build.gradle

```diff
@@ -19,11 +19,7 @@ ext.isReleaseVersion = !version.endsWith("SNAPSHOT")
 
 repositories {
     maven { url "https://plugins.gradle.org/m2/" }
```

Comment:
nit: can be replaced with `gradlePluginPortal()`

Reply:
Oh, I had meant to do that. I think I'll fix that later, though. I need this change to upgrade osdetector because kotlin.bintray.com's wedged state breaks the download process. So I'll also backport this to 0.9.x.

Reply:
Created https://github.com/google/protobuf-gradle-plugin/pull/666